WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -17,6 +17,10 @@ struct CollapsibleProductRowCard: View {
     //
     var shouldDisableDiscountEditing: Bool = false
 
+    // Tracks if discount should be allowed or disallowed.
+    //
+    var shouldDisallowDiscounts: Bool = false
+
```

Comment:
Perhaps this should be on the viewModel?

That way you can make a simple function/var on the view model which does the logic, rather than having the OR logic in the view.